### ISSUE TITLE
Error: No such command "init_db".

### ISSUE THREAD

**Issue by CrescentFresh** [aartur/mschematool#11]
Sometime overnight our CI began failing during database migrations at step `mschematool xyz init_db`:

`Error: No such command "init_db".`

Looking at the log of the last successful run and comparing it to the failing runs:

log from Successful job:
```
Collecting mschematool
  Downloading https://files.pythonhosted.org/packages/0f/ba/b0d8c564abbe2b8053b1c270de9d01668856074caee9b8f23466c726a73d/mschematool-0.9.tar.gz
Collecting click (from mschematool)
  Downloading https://files.pythonhosted.org/packages/34/c1/8806f99713ddb993c5366c362b2f908f18269f8d792aff1abfd700775a77/click-6.7-py2.py3-none-any.whl (71kB)
```

log from Failing job:
```
Collecting mschematool
  Downloading https://files.pythonhosted.org/packages/0f/ba/b0d8c564abbe2b8053b1c270de9d01668856074caee9b8f23466c726a73d/mschematool-0.9.tar.gz
Collecting click (from mschematool)
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
```

**Comment by CrescentFresh** [comment]
Issue reported to click issue tracker at https://github.com/pallets/click/issues/1123 Seems they done broke something overnight.

On a related note is there anyway to force mschematool to depend on a specific version of click, in this case the working v6.7?

**Comment by CrescentFresh** [comment]
For others reading we've worked around the issue with pip freeze, order of operations is important here (assuming fresh box with only python/pip installed):
```
$ pip install click==6.7
$ pip freeze
$ pip install mschematool
```
The result is that pip doesn't bother pulling down the latest click package, which would break mschematool.

**Comment by CrescentFresh** [comment]
See linked issue to click package repo, turns out it's due to the underscore _ in `init_db`. The intentional breaking change now require:

a) command authors (@aartur) are still expected to use underscore(s) in decorated method names
2) users (devs like me) are expected to use dashes (-) instead of underscores at the command line

They're working out something of a compromise for click v7.1 but v7.0 will forever break mschematool (and other packages that depend on click and use underscores in their method names).

So I guess the question is, can you lock mschematool tool to require click < v7.0 or >= v7.1?

**Comment by CrescentFresh** [comment]
Hello?

**Comment by CrescentFresh** [comment]
Hello?

**Comment by CrescentFresh** [comment]
Good lord what's a dev to do to get @aartur 's attention round here.

**Comment by CrescentFresh** [comment]
*crickets*

**Comment by CrescentFresh** [comment]
Bump

**Comment by CrescentFresh** [comment]
@aartur  @aartur  @aartur

**Comment by aartur** [comment]
I'm sorry guys, I thought I fixed the delivery of notifications but this was not the case (I think my email gateway's IPs are sometimes blocked by some ISPs).

I released 0.9.1 with package version pinned.

If someone is willing to take over the maintenance of `mschematool` I'll happily hand it over. I'm not using it for a few years already so problems that come up during normal usage are not visible to me.
